src/data-table.component.scss

```diff
@@ -13,15 +13,28 @@ $app-blue: rgba(48, 153, 197, 255);
 table {
```

Comment:
Since view encapsulation is now disabled in the component, all these rules should be namespaced (wrapped in the components selector)

Reply:
👍